kuberun/eventing/generic/main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START kuberun_eventing_generic_handler]
+
+// Sample generic is a Cloud Run service which logs and echos received requests.
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+// GenericHandler receives and echos a HTTP request's headers and body.
+func GenericHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Perhaps `EchoHandler` or just `Echo`? Generic is going to mean something else in Go relatively soon. :smiley:

Reply:
Thanks, I changed to EventHandler, also see my comment below.